docs/source/conf.py

```diff
@@ -51,53 +47,75 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path   = ['_static']
+html_theme         = 'bootstrap'
+html_favicon       = 'favicon.gif'
+html_logo          = 'favicon.gif'
+html_theme         = 'bootstrap'
+html_theme_options = {
+    'navbar_site_name': "Open MMBN",
+    'navbar_links': [
+        ("Contributing", "https://github.com/TheMaverickProgrammer/battlenetwork/wiki/Contributing", True)
+    ],
+    'source_link_position': "exclude",
+    'navbar_class': "navbar",
+    'navbar_fixed_top': "true",
+    'navbar_sidebarrel': False,  # Exhale does not support next / previous well
+    'bootswatch_theme': "sandstone",
+    'bootstrap_version': "3",
+}
 
 # -- Breathe setup -----------------------------------------------------------
-breathe_projects = { "C++ MMBN Project": "xml" }
-breathe_default_project = "C++ MMBN Project"
+# NOTE: see also, OUTPUT_DIRECTORY and XML_OUTPUT_DIRECTORY in Doxyfile.
+# If those change, this must update.
+this_file_dir = os.path.dirname(os.path.abspath(__file__))
+breathe_projects = {project: os.path.join(this_file_dir, '_doxygen', 'xml')}
+breathe_default_project = project
+
+# Tell sphinx not to process the doxygen output directory when searching for
+# files to render into html.
+exclude_patterns = ['_doxygen/**']
 
 # -- Exhale setup ------------------------------------------------------------
 exhale_args = {
-        "containmentFolder": "./api",
-        "rootFileName":      "library_root.rst",
-        "rootFileTitle":     "Code",
-        "doxygenStripFromPath": "../../BattleNetwork",
-        "createTreeView":    True,
-        "exhaleExecutesDoxygen": True,
-        # "exhaleDoxygenStdin": "INPUT = ../../BattleNetwork",
-        "exhaleUseDoxyfile": True,
-        "unabridgedOrphanKinds": {"dir", "file" }
-        }
+    "containmentFolder": "./api",
+    "rootFileName": "library_root.rst",
+    "rootFileTitle": "Code",
+    # NOTE: see STRIP_FROM_PATH in Doxyfile.  If that updates, this must too.
+    "doxygenStripFromPath": "../../BattleNetwork",
+    "fullToctreeMaxDepth": 1,
+    # See edits in source_read.  This tricks Exhale into not including a
+    # top-level "Full API" heading on the unabridged document.
+    "fullApiSubSectionTitle": "",
+    "createTreeView": False,  # See source_read, they get pruned.
+    "exhaleExecutesDoxygen": True,
+    "exhaleUseDoxyfile": True,
+    "unabridgedOrphanKinds": {"dir"}
```

Comment:
Combo of `fullToctreeMaxDepth` + `createTreeView` + `fullApiSubSectionTitle` changes here and `source_read` hacks below push the API page closer to what I think you probably want.  The `createTreeView = False` given that we filter out the `.. include::` directives in `source_read` is just to avoid adding any extra CSS / JavaScript.  Since I'm deleting the class / file hierarchies, the CSS / JS aren't needed.

Note that I'm setting `unabridgedOrphanKinds` to `{"dir"}` because now that we're deleting the file hierarchy, the files now show up at the bottom making it less obtrusive, but still linked.  It's basically the reverse of what you asked for lol, but something tells me you'd rather have the files pushed to the bottom rather than the top.  The current `STRIP_FROM_PATH` and `doxygenStripFromPath` are chosen to make it so that file destinations are what would be `#include`d, but that means no top-level directory, which means file hierarchy is long and dense.